modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -95,13 +101,19 @@ pullSecret: '<pull_secret>' <6>
 Do not move the `install-config.yaml` and `agent-config.yaml` configuration files to the `openshift` directory.
 ====
 
-. Complete the steps in the link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files["Configuration Files"] section of the _Oracle_ documentation to download {oci-ccm-full} and {oci-csi-full} manifests as an archive file and save the archive file in your `openshift` directory. You need the {oci-ccm} manifests for deploying the {oci-ccm} during cluster installation so that {product-title} can connect to the external {oci} platform. You need the {oci-csi} custom manifests for deploying the {oci-csi} driver during cluster installation so that {product-title} can claim required objects from {oci}.
+. Download and prepare custom manifests for Oracle:
+
+.. Go to link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files[Configuration Files] (Oracle documentation).
 
-. Access the custom manifest files that are provided in the link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files["Configuration Files"] section of the _Oracle_ documentation.
+.. Download the {oci-ccm-full} and {oci-csi-full} manifests as an archive file and save the archive file in your `openshift` directory.
 +
+You need the {oci-ccm} manifests for deploying the {oci-ccm} during cluster installation so that {product-title} can connect to the external {oci} platform. You need the {oci-csi} custom manifests for deploying the {oci-csi} driver during cluster installation so that {product-title} can claim required objects from {oci}.
+
+.. Access the {oci-ccm} and {oci-csi} manifests that you downloaded.
+
 .. Change the `oci-cloud-controller-manager` secret that is defined in the `oci-ccm.yml` configuration file to match your organization's region, compartment {ocid}, VCN {ocid}, and the subnet {ocid} from the load balancer.
 
-. Use the Agent-based Installer to generate a minimal ISO image, which excludes the rootfs image, by entering the following command in your {product-title} CLI. You can use this image later in the process to boot all your cluster’s nodes.
+. Generate a minimal ISO image, which excludes the rootfs image, by entering the following command. You can use this image later in the process to boot all your cluster's nodes.
```

Comment:
Similar to my other question - if adding the installation directory step was valid, should this command have an argument to specify that dir? Or, alternatively, should I mention in the wording that you need to run this in your installation directory?